src/provider/rainbowKit/chainConfig.ts

```diff
@@ -0,0 +1,170 @@
+import { CarbonEvmChainIDs, Network, NetworkConfigs } from "@carbon-sdk/constant";
+import { parseChainId } from "@carbon-sdk/util/ethermint";
+
+export interface ChangeNetworkParam {
```

Comment:
i feel these values could be refactored into a new chainConfig.ts file under the constants folder? since MetaMask.ts uses the exact config